cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -47,7 +48,9 @@ var CMS = window.CMS || {};
             if (++clickCounter === 1) {
                 timer = setTimeout(function () {
                     clickCounter = 0;
-                    window.location.href = $(e.currentTarget).attr('href');
+                    anchor = $(e.currentTarget);
```

Comment:
I'm not sure, we do the same with timer so it might be better to also cache anchor, what you think?